components/MaterialsForm/Machines/index.js

```diff
@@ -43,11 +65,21 @@ export default function Machines() {
             <option value="Laubbläser">Laubbläser</option>
             <option value="Kompaktbagger">Kompaktbagger</option>
           </StyledSelect>
-          {newMachines.length - 1 === index && (
-            <StyledAddButton type="button" onClick={handleAddNewMachines}>
-              <span>+</span>
-            </StyledAddButton>
-          )}
+          <StyledButtonContainer>
+            {!hideButton && (
+              <StyledDeleteButton
+                type="button"
+                onClick={() => handleDeleteMachines(index)}
+              >
+                <span>X</span>
+              </StyledDeleteButton>
+            )}
+            {newMachines.length - 1 === index && (
+              <StyledAddButton type="button" onClick={handleAddNewMachines}>
+                <span>+</span>
+              </StyledAddButton>
+            )}
```

Comment:
The buttons should have an `aria-label` so that the screen reader can tell the user what they are for.
Besides, I think you can remove the `span` element.

Reply:
Thanks! I'll do it immediately.